core/core.podspec

```diff
@@ -1,14 +1,14 @@
 Pod::Spec.new do |spec|
     spec.name                     = 'core'
-    spec.version                  = '1.0'
+    spec.version                  = '1.0.0-SNAPSHOT'
     spec.homepage                 = ''
     spec.source                   = { :http=> ''}
     spec.authors                  = ''
     spec.license                  = ''
     spec.summary                  = ''
     spec.vendored_frameworks      = 'build/cocoapods/framework/Purchases.framework'
     spec.libraries                = 'c++'
-    spec.ios.deployment_target = '11.0'
+    spec.ios.deployment_target = '13.0'
```

Comment:
Ohh this should have been part of the PHC 13 bump I imagine... Do we need to update `main` with this change?

Reply:
This is on `main` already afaik. I just set the correct value in the `cocoapods { }` block in the `build.gradle.kts`. This `.podspec` file is auto-generated from that `cocoapods { }` block.